src/App.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Signup from "./pages/Signup";
+import Login from "./pages/Login";
+import Home from "./pages/Home";
+import { auth } from "../src/utils/firebase/firebase.utils";
+import ProtectedRoute from "./components/ProtectedRoute";
+
+function App() {
+  const [isAuth, setIsAuth] = useState(false);
+  useEffect(() => {
+    auth.onAuthStateChanged((user) => {
+      if (user) {
+        setIsAuth(true);
+      }
+    });
+  }, []);
+
+  return (
+    <div className="App">
+      <Router>
+        <Routes>
+          <Route path="/home" element={<Home />} />
+          <Route path="/" element={<Signup />} />
```

Comment:
You should write your base path, i.e. '/', above your other paths ('/home/ and '/login')